presto-spi/src/main/java/com/facebook/presto/spi/function/CodegenScalarFunction.java

```diff
@@ -24,6 +26,28 @@
 @Target(METHOD)
 public @interface CodegenScalarFunction
 {
+    class CodegenMethodHandles
```

Comment:
nits: Let's just call this `CodegenMethodHandle` without the `s`. It was a bit confusing to me why there are multiple method handles for a single function.